src/vue-pdf-embed.vue

```diff
@@ -215,6 +226,7 @@ export default {
     async renderPage(page, canvas, width) {
       const viewport = page.getViewport({
         scale: Math.ceil(width / page.view[2]) + 1,
+        rotation: this.rotate % 90 === 0 ? this.rotate : 0,
```

Comment:
Instead of calculating a value for each page/layer, let's maybe use a [validator function](https://vuejs.org/guide/components/props.html#prop-validation) to ensure that `this.rotate` is `undefined`, `0` or a multiple of `90`?